integration-tests/src/test/java/oracle/kubernetes/operator/ITPodsRestart.java

```diff
@@ -0,0 +1,212 @@
+
+    logger.info(
+        "About to testDomainServerPodRestart for Domain: "
+            + domain.getDomainUid()
+            + "  logHomeEnabled: true -->  logHomeEnabled: false");
+    domain.testDomainServerPodRestart("logHomeEnabled: true", "logHomeEnabled: false");
+
+    logger.info("SUCCESS - " + testMethodName);
+  }
+
+  /**
+   * The property tested is: imagePullPolicy: IfNotPresent --> imagePullPolicy: Never
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testServerPodsRestartByChangingImagePullPolicy() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+
+    boolean testCompletedSuccessfully = false;
+
+    logger.info(
+        "About to testDomainServerPodRestart for Domain: "
+            + domain.getDomainUid()
+            + " imagePullPolicy: IfNotPresent -->  imagePullPolicy: Never ");
+    domain.testDomainServerPodRestart(
+        "imagePullPolicy: \"IfNotPresent\"", "imagePullPolicy: \"Never\" ");
+
+    logger.info("SUCCESS - " + testMethodName);
+  }
+
+  /**
+   * The property tested is: includeServerOutInPodLog: true --> includeServerOutInPodLog: false
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testServerPodsRestartByChangingIncludeServerOutInPodLog() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+
+    boolean testCompletedSuccessfully = false;
+
+    logger.info(
+        "About to testDomainServerPodRestart for Domain: "
+            + domain.getDomainUid()
+            + "  includeServerOutInPodLog: true -->  includeServerOutInPodLog: false");
+    domain.testDomainServerPodRestart(
+        "includeServerOutInPodLog: true", "includeServerOutInPodLog: false");
+
+    logger.info("SUCCESS - " + testMethodName);
+  }
+
+  /**
+   * The property tested is: image: "store/oracle/weblogic:12.2.1.3" --> image:
+   * "store/oracle/weblogic:duplicate"
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testServerPodsRestartByChangingImage() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+
+    boolean testCompletedSuccessfully = false;
+    try {
+      logger.info(
+          "About to testDomainServerPodRestart for Domain: "
+              + domain.getDomainUid()
+              + "  Image property: store/oracle/weblogic:12.2.1.3 to store/oracle/weblogic:duplicate");
+      TestUtils.dockerTagImage("store/oracle/weblogic:12.2.1.3", "store/oracle/weblogic:duplicate");
+      domain.testDomainServerPodRestart(
+          "\"store/oracle/weblogic:12.2.1.3\"", "\"store/oracle/weblogic:duplicate\"");
+    } finally {
+      TestUtils.dockerRemoveImage("store/oracle/weblogic:duplicate");
```

Comment:
replace with TestUtils.exec("docker rmi -f store/oracle/weblogic:duplicate")

Reply:
Almost missed it. Done